packages/kit/src/runtime/client/start.js

```diff
@@ -19,6 +19,12 @@ export async function start({ env, hydrate, paths, target, trailing_slash }) {
 	set_public_env(env);
 	set_paths(paths);
 
+	if (__SVELTEKIT_DEV__ && target === document.body) {
+		console.warn(
+			`Placing %sveltekit.body% directly inside <body> is not recommended, as your app may break for users who have certain browser extensions installed.\n\nConsider wrapping it in an element:\n\n<div style="display: contents">\n  %sveltekit.body%\n</div>`
+		);
+	}
```

Comment:
Do we want to coordinate anything with the documentation at https://kit.svelte.dev/docs/project-structure#project-files-src ? Should this message link you to any docs? Should the docs say that this will produce a warning?

Reply:
updated the docs, but i don't think it's worth linking to them because they don't provide any additional information